ccd-definition/AuthorisationCaseEvent/systemUpdate.json

```diff
@@ -5,5 +5,12 @@
     "CaseEventID": "MOVE_TO_STAYED",
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "NOTIFY_DEFENDANT_SOLICITOR_FOR_CLAIM_ISSUE",
```

Comment:
does this need to specify which defendant solicitor?

Reply:
will need to speak to Allison for all event naming convention, however this is just place holder